piechart/piechart.go

```diff
@@ -7,15 +7,13 @@
 package piechart
 
 import (
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
```

Comment:
You're right about this, I need to change that but if you don't mind, I'll do it separately from this PR (at the same time I'll add a `go.mod` file to this package, too).

Could you leave it unchanged?

Reply:
ok